hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/compactions/Compactor.java

```diff
@@ -115,7 +115,7 @@ public CompactionProgress getProgress() {
   }
 
   /** The sole reason this class exists is that java has no ref/out/pointer parameters. */
-  protected static class FileDetails {
+  public static class FileDetails {
```

Comment:
why the change to public? it looks like all current uses should be fine with protected.